src/app/app.component.scss

```diff
@@ -24,6 +24,8 @@ ion-menu.md ion-list-header {
   padding-right: 18px;
   padding-left: 18px;
 
+  font-size: min(1.375rem, 36px);
```

Comment:
I would have preferred to truncate instead of adjusting the font size, but there's no shadow part for the element I would need to target to truncate.